File: public/code/compilers.py
# Create a Jutge API Client
from jutge_api_client import JutgeApiClient
jutge = JutgeApiClient()

# Get all compilers and print their name and programming language
compilers = jutge.tables.get_compilers()
for id, compiler in compilers.items():
    print(f"{id}: '{compiler.name}' ({compiler.language})")

File: public/code/compilers.ts
// Import the Jutge API client and its extras
import { JutgeApiClient } from './jutge_api_client'

// Make a client
const jutge = new JutgeApiClient()

// Get all compilers
const compilers = await jutge.tables.getCompilers()

// Filter C++ compilers amd print their name
for (const [id, compiler] of Object.entries(compilers)) {
    console.log(`${id}: ${compiler.name} (${compiler.language})`)
}
